subr/check_version.sh

```diff
@@ -0,0 +1,8 @@
+export selfupdate_macports="false"
+export macports_version="$1"
+
+if [ "$1" = "" ]; then
+	macports_release_url=$(curl https://raw.githubusercontent.com/macports/macports-base/master/config/RELEASE_URL || curl https://trac.macports.org/export/HEAD/macports-base/config/RELEASE_URL || curl https://distfiles.macports.org/MacPorts/RELEASE_URL)
+	export macports_version="$(echo $macports_release_url | awk -F'/v' '{ print $NF }')"
+	export selfupdate_macports="true"
```

Comment:
Attention to the intent conveyed by quotes. Double quotes are for interpolation and single quotes for literal expressions. It's a bit weird to write `"true"` rather than `true` here.